index.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Portfolio page</title>
+	<meta name="description" content="Holly's portfolio page">
+	<meta name="author" content="Holly Bedford">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css" type="text/css">
+	<link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css" type="text/css">
+</head>
+
+<body>
+	<header>
+		<a class="btn" href="#contacts">Contact</a>
+		<h3>Portfolio page</h3>
+		<a class="btn" href="#items">Portfolio stuff</a>
+		
+	</header>
+	
+	<div class="banner"></div>
+	
+	<main>
+		<p class="quote">"Spends too much time with the cat"</p>
+		<a class="btn" href="#about-me">About me</a>
+	</main>
+	
+	<div class="banner">Portfolio Stuff</div>
+	
+	<section class="items" id="items">
+		<div class="portfolio-item pi-image1">
+			<h5>Podcast editing skills</h5>
+		</div>
+		<div class="portfolio-item pi-image2">
+			<h5>Portfolio wireframe</h5>
+			<p>I used Balsamiq to do the wireframe as it's low-fidelity and all I needed at the time.</p>
+		</div>
+		<div class="portfolio-item pi-image3">
+			<h5>Portfolio item here</h5>
+			<p class="no">At some point there will be another portfolio item here but at the moment there is nothing interesting.<br>Or is there?</p>
+		</div>
+		<div class="portfolio-item pi-image4" id="flower-image">
+			<h5>Flower arranging</h5>
+		</div>
+	</section>
+	<div class="banner" >Things to know</div>
+	<section id="about-me" class="about-me">
+		<h3>About me</h3>
+		<p>What's this you've said to me, my good friend? I'll have you know I graduated top of my class in conflict resolution, and I've been involved in numerous friendly discussions, and I have over 300 confirmed friends. I am trained in polite discussions and I'm the top mediator in the entire neighborhood. You are worth more to me than just another target. I hope we will come to have a friendship never before seen on this Earth.</p>
+	</section>
+	
+	
+	<div class="banner" >Contact Stuff</div>
+	<div class="contacts" id="contacts">
+		<a class="social" href="https://github.com/hb020202/Portfolio" target="_blank">
+			<img class="github" src="images/github.png" alt="github logo">
+		</a>
+		<p class="social">holly.bedford02@gmail.com</p>
```

Comment:
Totally optionally, but if you wanted to make your email a clickable <a> tag so when clicked, it opens up a user's email client with the email in the 'to' field, you can do something like this, adding the 'mailto:' in front of your email address!

`<a href="mailto:webmaster@example.com">Jo Do</a>`